File: 0167-two-sum-ii-input-array-is-sorted/0167-two-sum-ii-input-array-is-sorted.js
/**
 * @param {number[]} numbers
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(numbers, target) {
    let i=0;
    let j=numbers.length-1;
    while(i<j){
        let su=numbers[i]+numbers[j]
        if(su>target){
            --j
        }else if(su<target){
            ++i
        }else{
            return [i+1,j+1]
        }
   

    }
     return [-1]
};

File: 0167-two-sum-ii-input-array-is-sorted/0167-two-sum-ii-input-array-is-sorted.py
class Solution(object):
    def twoSum(self, numbers, target):
        """
        :type numbers: List[int]
        :type target: int
        :rtype: List[int]
        """
        left=0
        right=len(numbers)-1
        while left<right:
            curr_sum=numbers[left]+numbers[right]
            if curr_sum==target:
                return [left+1,right+1]
            elif curr_sum>target:
                right-=1
            else:
                left+=1
        